include/openmc/string_utils.h

```diff
@@ -15,13 +17,23 @@ std::string to_element(const std::string& name);
 
 void to_lower(std::string& str);
 
-int word_count(std::string const& str);
+int word_count(const std::string& str);
 
 vector<std::string> split(const std::string& in);
 
 bool ends_with(const std::string& value, const std::string& ending);
 
 bool starts_with(const std::string& value, const std::string& beginning);
 
+template<typename T>
+inline std::string concatenate(const T& temps)
+{
+  std::ostringstream oss;
+  for (const auto& temp : temps) {
+    oss << temp << " ";
```

Comment:
This adds a little complexity, but again because this function could be used for a variety of types, it would be nice to include a `delimiter` parameter that defaults to `" "` to allow one to customize the format a bit.

Reply:
Solved. By default the delimiter is " ", but in src/nuclide.cpp, src/thermal.cpp and src/mgxs.cpp ", " is used.